entity-service-change-event-generator/src/main/java/org/hypertrace/entity/service/change/event/impl/EntityChangeEventGeneratorImpl.java

```diff
@@ -0,0 +1,123 @@
+package org.hypertrace.entity.service.change.event.impl;
+
+import static java.util.function.Function.identity;
+
+import com.google.common.collect.MapDifference;
```

Comment:
Make sure to declare guava dep (likely getting it transitive as a grpc dependency right now, which we want to be explicit about)